.formatter.exs

```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
+[
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
```

Comment:
nice, dunno why this was not part of the lib, maybe you should import ecto config to prevent formating link `drop_if_exists(table(:users))`

```suggestion
  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"],
  import_deps: [:ecto],
```